### DISCUSSION TITLE
Import multiple meetings - not picking up day or time

### DISCUSSION THREAD

**Discussion by Goonysings** [code4recovery/12-step-meeting-list#557] [Q&A] [answered]
I'm sure this is a user error. My spreadsheet shows a time and day for each meeting and the import is changing to "appointment". These are multiple meetings at the same location. I changed the slugs to a -numeric and the import still shows appointment. Is there a work-around
[District70_meetingslist.csv](https://github.com/code4recovery/12-step-meeting-list/files/7370001/District70_meetingslist.csv)
?

**Comment by joshreisner** [comment]
Hm, seems like an issue! It happened to me importing on [our demo site](https://demo.code4recovery.org/meetings?tsml-day=any).

Also I don't see "Appointment" in the time field, and I see "undefined" when I use the page controls.

**Reply by tech2serve** [reply]
I imported our `template.csv` without any issues. When I imported the attached csv, I got "Appointment" or day and time. 

I suspect the issue lies in the file which I'm looking at closer now.

**Comment by tech2serve** [comment, marked as answer]
Hi @Goonysings,

The problem appears to be with the data in the`updated` column. I’m not sure exactly why, but I was able to successfully import your data by completely either of the following steps.

1. Delete the column.
2. Replace the column with just a date.

In either case, TSML used the time stamp of the import for the publish data, so deleting it in your spreadsheet is safe.

Tim

**Reply by joshreisner** [reply]
huh, that's a good catch!

**Reply by Goonysings** [reply]
This didn't solve my issue :( 
<img width="953" alt="Screen Shot 2021-10-26 at 4 18 46 PM" src="https://user-images.githubusercontent.com/92771252/138962502-9d099c46-5d1a-4ec8-abc3-e5866ec07cff.png">

**Reply by Goonysings** [reply]
I downloaded this and entered the times on the new sheet for the first 7 meetings and it uploaded correctly. I'll use this sheet to build out the rest of the district. 

Thanks all!
